util/logging.go

```diff
@@ -42,6 +53,16 @@ var (
 	// All go-vcloud director logging goes through this logger
 	GovcdLogger *log.Logger
 
+	// HTTP operation stack
+	GovcdHttpStack FixedSizeStack
```

Comment:
I think you should add a more descriptive comment or explain what this is and why it is useful in logging.md, or both.

Reply:
Will do, and add some more info in TESTING.md